python/lsst/dax/imgserv/imageREST_v0.py

```diff
@@ -226,6 +233,14 @@ def getIDeepCoaddCutout():
 def getIDeepCoaddCutoutPixel():
     return _getICutout(request, W13DeepCoaddDb, 'pixel')
 
+# this will handle something like:
+# GET /image/v0/deepCoadd/cutout?ra=19.36995&dec=-0.3146widthAng=30.0&heightAng=45.0
+# GET /image/v0/deepCoadd/cutout?ra=19.36995&dec=-0.3146xx&widthPix=100&heightPix=100
```

Comment:
Technically, these are too long, but I don't like the idea of splitting the example URL across 2 lines.